api/handlers/user.js

```diff
@@ -398,20 +398,20 @@ var UserHandler = {
         if (!utils.validateLeadersInputs(req)) {
           return res.status(412).send('Invalid inputs');
         }
-
         var categoryName = req.params.categoryName;
         var expert = req.query.expert === '1';
+        var query;
+        if (expert) {
```

Comment:
The leaders function was pretty broken. This mongo query takes all the logic out of the server